File: BasicRxJavaSample/app/src/main/java/com/example/android/observability/demoTest/ImitateUser.java
package com.example.android.observability.demoTest;

import androidx.annotation.NonNull;
import androidx.room.ColumnInfo;
import androidx.room.Entity;
import androidx.room.Ignore;
import androidx.room.PrimaryKey;

import java.util.UUID;

@Entity(tableName = "Users")
public class ImitateUser {
    @NonNull
    @PrimaryKey
    @ColumnInfo(name = "userId")
    private String mId;

    @ColumnInfo(name = "userName")
    private String mUserName;

    public ImitateUser(String userName){
        mId = UUID.randomUUID().toString();
        mUserName = userName;
    }

    public ImitateUser(@NonNull String userId, String userName){
        mId = userId;
        mUserName = userName;
    }

    public String getId() {
        return mId;
    }

    public String getUserName() {
        return mUserName;
    }

    public void setId(@NonNull String mId) {
        this.mId = mId;
    }

    public void setUserName(String mUserName) {
        this.mUserName = mUserName;
    }
}

File: BasicRxJavaSampleKotlin/app/src/main/java/com/example/android/observability/imitate/persistance/ImitateUser.kt
package com.example.android.observability.imitate.persistance

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey
import org.jetbrains.annotations.NotNull
import java.util.*

@Entity(tableName = "users")
data class ImitateUser(
        @PrimaryKey
        @ColumnInfo(name="userId")
        val userId:String=UUID.randomUUID().toString(),
        @ColumnInfo(name="userName")
        val  name:String) {
}